socorro/unittest/conftest.py

```diff
@@ -30,163 +30,7 @@ def test_something(metricsmock):
     return MetricsMock()
 
 
-class DjangoTablesMixin(object):
```

Comment:
I literally just wrote this earlier this week, so I'm a bit bummed to see it go so soon. But ... so it goes.